### PR TITLE
Fix HH Bribing and Claiming scripts

### PR DESCRIPTION
**Pull request by sajanrajdev**
Tackles https://github.com/Badger-Finance/badger-multisig/issues/1335

### CODE REVIEW

**Review by petrovska-petro — COMMENTED** [review]
updates addrs lgtm and interfaces, abi changes are minimal in the methods ✅ 

@saj, could we make more flexible the `periods` and `maxTokensPerVote` values instead of hardcoded (by default `_maxTokensPerVote=0` & `_periods=1` 

i think the most value on their upgrade lies into those params. while we may do one per period and accepting whatever amounts of votes/token, i will say that having that dynamic as value accepted on the cli will be useful from the influence strat pov

**Comment by sajanrajdev** [comment]
> updates addrs lgtm and interfaces, abi changes are minimal in the methods ✅
> 
> @saj, could we make more flexible the `periods` and `maxTokensPerVote` values instead of hardcoded (by default `_maxTokensPerVote=0` & `_periods=1`
> 
> i think the most value on their upgrade lies into those params. while we may do one per period and accepting whatever amounts of votes/token, i will say that having that dynamic as value accepted on the cli will be useful from the influence strat pov

Good call, will do so.

**Comment by sajanrajdev** [comment]
@petrovska-petro, introduced the ability to choose the period and bribes per period via function parameters. In addition, I added a check for 0 bribes on BADGER/wBTC for Balancer as it is the case for this round, attempting to bribe with 0 reverts as expected.

**Review by petrovska-petro — APPROVED** [review]
allows flex for the case that influence pod decides to be more granular with rounds or expected votes per incentive amts ✅ 

addrs in book are correct ✅ 

checks if when passing concatenated incentives for bal hh market any of them are `0` to skipped them ✅

File: scripts/badger/bribe_ecosystems.py [hunk 1]
```diff
@@ -59,6 +59,8 @@ def main(
     badger_bribe_in_votium=0,
     badger_bribe_in_frax=0,
     badger_bribe_in_bunni=0,  # NOTE: dollar denominated. Badger calculation is done internaly
+    max_tokens_per_vote=0,  # Maximum amount of incentives to be used per round (Hidden Hands V2)
+    periods=1,  # Rounds to be covered by the incentives deposited (Hidden Hands V2)
```
Comment (gosuto-inzasheru) [inline scripts/badger/bribe_ecosystems.py:63]: in cases such as these, it is usually better to add new params to the end of the signature, to maintain backwards compability.

**Review by gosuto-inzasheru — APPROVED** [review]
confirmed onchain with command `brownie run badger/bribe_ecosystems main 0 0,16000`. runs successfully

**Pull request merged**